examples/canoniCAI/data/ner_gen

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
What is this file for? Do we need to commit this?

Reply:
this used to auto ner json template based on the ner training data. Because we have no frontend which will tell us the start index and end index of a slot and generate it to a json. I used it there to help me generate a simple json for single slot cases for utterances in the training set.

Reply:
Rename this to `ner_gen.py` and add at the top of this file a comment block explaining what this file is and how to use it.

"""
This file is for blah blah blah.
You can run it with Blah blah balh
"""